src/cr/cube/crunch_cube.py

```diff
@@ -405,7 +406,7 @@ def _margin(self, axis=None, weighted=True, adjusted=False,
 
         return res
 
-    def _mr_proportions(self, axis, weighted):
+    def _mr_proportions(self, axis, weighted, include_transforms_for_dims=None):
         '''Calculate MR proportions.'''
```

Comment:
include_transforms_for_dims is starting to feel like something we need to factor out.

Reply:
Created a ticket for this: https://www.pivotaltracker.com/story/show/156313137